submissions/IRETIOGEDENGBE/database/database.txt

```diff
@@ -0,0 +1 @@
+https://docs.google.com/document/d/1iB0WyeQxSdzAPpmzycSiS68QH3c6KWDY24HVUlF63ts/edit
```

Comment:
Nice attempt, Ireti.
On the inner join exercise, I see that you are joining on `brands.id=products.id`, this should be on `brands.id=products.brand_id` because brand_id is the foreign key to brands on the products table